osu.Game.Rulesets.Osu/Difficulty/Evaluators/FlashlightEvaluator.cs

```diff
@@ -52,23 +54,36 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool hidd
                 var currentObj = (OsuDifficultyHitObject)current.Previous(i);
                 var currentHitObject = (OsuHitObject)(currentObj.BaseObject);
 
+                cumulativeStrainTime += lastObj.StrainTime;
+
                 if (!(currentObj.BaseObject is Spinner))
                 {
-                    double jumpDistance = (osuHitObject.StackedPosition - currentHitObject.StackedEndPosition).Length;
+                    double pixelJumpDistance = (osuHitObject.StackedPosition - currentHitObject.StackedEndPosition).Length;
```

Comment:
`pixelJumpDistance` was named as such to match `pixelTravelDistance`, since the evaluator relies on pixel based distances rather than the radius normalised ones. I would say removing jump to make `pixelDistance` is fine.